static/js/popovers.js

```diff
@@ -1184,4 +1184,17 @@ exports.compute_placement = function (elt, popover_height, popover_width,
     return placement;
 };
 
+// Generic tooltip to replace HTML titles.
+exports.create_generic_tooltip = function (elem, title, placement) {
```

Comment:
I put this in `popovers.js` temporarily as I'm not sure where to finally move this to.

@timabbott Suggestions on where to move this?

Reply:
How about `ui.js`?

Reply:
I've moved it to `ui.js`. I also added a commit to replace the HTML titles with static text in message controls.